src/components/content/content-theme.scss

```diff
@@ -1,6 +1,4 @@
 md-content.md-THEME_NAME-theme {
   color: '{{foreground-1}}';
-  background-color: '{{background-default}}';
+  background-color: '{{background-color}}';
```

Comment:
Indeed, I was just wondering where the `color` suffix is validated in the theming service.